plugins/BEdita/Core/tests/TestCase/Model/Entity/ObjectEntityTest.php

```diff
@@ -506,14 +507,14 @@ public function testGetRelationshipsDeleted()
      */
     public function testGetRelationshipsIncluded()
     {
-        $entity = TableRegistry::get('Documents')->get(2, ['contain' => ['Test']]);
+        $entity = TableRegistry::get('Folders')->get(12, ['contain' => ['Parents']]);
```

Comment:
[very minor] Instead of editing this test case, I'd prefer adding a new one.

Reply:
ok, done!